apic_ml2/neutron/plugins/ml2/drivers/cisco/apic/network_constraints.py

```diff
@@ -35,6 +35,10 @@ def get_subnet_constraints(self, tenant, network):
         """Returns (default-scope, scope-for-tenant, scope-for-network) """
         return (None, None, None)
 
+    def get_allow_vm_names(self, tenant, vrf):
```

Comment:
nit: called this 'get_allowed_vm_names'. Also, I would think that the arguments to this function would be OS tenant and OS network